client/components/fields/editor/EventRelatedPlannings/RelatedPlanningItem.tsx

```diff
@@ -133,12 +131,25 @@ export class RelatedPlanningItem extends React.PureComponent<IProps> {
                                     'planning',
                                     item as IPlanningItem,
                                     (item) => {
-                                        this.update(item);
-
-                                        return Promise.resolve(item);
+                                        // Remove fields responsible for creating a link, so that item can be
+                                        // standalone until user decides to link it through main editor "Save"
+
+                                        const itemClean = omit(
+                                            modifyForServer(item, true),
+                                            ['_temporary', 'related_events', '_created', '_etag', '_links', '_updated'],
```

Comment:
Ensure field names are typed.  It be best to move [this](https://github.com/superdesk/superdesk-client-core/blob/develop/scripts/core/helpers/utils.tsx#L39) to `@sourcefabric/common` and use here